website/src/constants/docsConstants.js

```diff
@@ -69,6 +69,7 @@ export const PROPS_TABLE_CALENDAR_ROWS = [
     they can't be included in a day range. If user tries to select/include them onDisabledDayError will be called`
   ],
   ['onDisabledDayError', 'Function', 'disabledDay => null', 'Gets called when user tries to select/include a disabled day'],
+  ['disableBackwards', 'Boolean', 'false', 'If true, disable all days before today'],
```

Comment:
Can you remove this for now, please? I will add this to docs as of version 1.1.0(next version including year/month select feature)

Reply:
No problem, I will.